packages/api-tests/reporter.js

```diff
@@ -0,0 +1,62 @@
+var mocha = require('mocha');
+module.exports = MyReporter;
+const testContainer = process.env.MOCHA_CONTAINER;
+var fs = require('fs');
+var path = require('path');
+
+let jsonOutput = [];
+const pattern = /(#[\w.()]+[\s]?)+/;
```

Comment:
The whitespace is needed in case we have multiple tags. If we only want max 1 tag per method then I can remove it?